Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -252,10 +287,18 @@ int RGroupDecomposition::add(const ROMol &inmol) {
       }
     }
   }
-
-  if (rcore == nullptr) {
-    BOOST_LOG(rdDebugLog) << "No core matches" << std::endl;
-    return -1;
+  // mark any wildcards in input molecule:
+  for (auto &atom : mol.atoms()) {
+    if (atom->getAtomicNum() == 0) {
+      atom->setProp(_rgroupInputDummy, true);
+      // clean any existing R group numbers
+      atom->setIsotope(0);
+      atom->setAtomMapNum(0);
+      if (atom->hasProp(common_properties::_MolFileRLabel)) {
+        atom->clearProp(common_properties::_MolFileRLabel);
+      }
```

Comment:
I know this isn't your code, but we might as well clean this up at the same time:
```suggestion
      atom->clearProp(common_properties::_MolFileRLabel);
```

Reply:
Done, thanks for noticing!